public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -1267,6 +1267,10 @@ export class DashboardModel implements TimeModel {
     return Boolean(this.meta.canEdit || this.meta.canMakeEditable);
   }
 
+  canExecuteActions() {
+    return this.canEditDashboard();
```

Comment:
This seems reasonable/safe for a first pass; but eventually I suspect we will want more 😛

Reply:
yes, long term we'll add a dedicated permission 😁